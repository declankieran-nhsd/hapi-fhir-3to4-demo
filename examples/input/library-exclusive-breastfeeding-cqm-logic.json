{
  "resourceType": "Library",
  "text": {
    "status": "generated",
    "div": "<div>Exclusive Breastfeeding CQM Logic</div>"
  },
  "moduleMetadata": {
    "identifier": [
      {
        "use": "official",
        "value": "Exclusive_Breastfeeding_CQM_Logic"
      }
    ],
    "version": "1.0.0",
    "title": "Exclusive Breastfeeding CQM Logic",
    "type": "library",
    "status": "active",
    "experimental": true,
    "description": "Quality measure logic for measuring outcomes for exclusive breastmilk feeding of newborns",
    "publicationDate": "2016-03-08",
    "topic": [
      {
        "text": "Exclusive Breastfeeding"
      }
    ]
  },
  "model": [
    {
      "identifier": "QUICK"
    }
  ],
  "valueSet": [
    {
      "name": "Female Administrative Sex",
      "identifier": "2.16.840.1.113883.3.560.100.2"
    }
  ],
  "dataRequirement": [
    {
      "fhir_comments": [
        "    ... other value sets as defined in the library    "
      ],
      "type": "Condition",
      "codeFilter": [
        {
          "path": "code",
          "valueSetString": "Single Live Birth"
        }
      ]
    }
  ],
  "document": {
    "fhir_comments": [
      "    ... other data requirements as defined in the library    "
    ],
    "contentType": "application/cql+text",
    "url": "http://cqlrepository.org/CMS9v4_CQM.cql"
  }
}