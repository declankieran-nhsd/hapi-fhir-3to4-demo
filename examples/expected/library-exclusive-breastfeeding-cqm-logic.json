{
  "resourceType": "Library",
  "text": {
    "status": "generated",
    "div": "<div>Exclusive Breastfeeding CQM Logic</div>"
  },
  "dataRequirement": [
    {
      "type": "Condition",
      "codeFilter": [
        {
          "path": "code",
          "valueSetString": "Single Live Birth"
        }
      ]
    }
  ]
}